reference/constraints/Valid.rst

```diff
@@ -261,8 +261,8 @@ traverse
 
 **type**: ``boolean`` **default**: ``true``
 
-If this constraint is applied to a property that holds an array of objects,
-then each object in that array will be validated only if this option is
-set to ``true``.
+If this constraint is applied to a ``Traversable``, then all containing values
+will be validated if this option is set to ``true``. This option is ignored on
+arrays: Arrays are traversed in either case. Keys are not validated.
```

Comment:
This should need a correction if https://github.com/symfony/symfony/pull/29800 is merged, right?

Reply:
No, `@Valid` is working correctly. The docs could just use this clarification. The mentioned pull request is only concerned with other constraints not working correctly.